doc/baremetal/hostlink.rst

```diff
@@ -21,11 +21,15 @@ a need to provide implementation of ``_exit()`` either through linking with some
 libgloss implementation or by implementing it right in the application.
 
 Because libgloss implementation is often specific to a particular chip (nee
```

Comment:
I suspect it was intended this way, because "see" assumes there should be page about "BSP", while "nee" means "an original name of something" - hence implication that libgloss implementation is itself the BSP. But the choice of word is weird to communicate the idea that libgloss == BSP, not sure why I did it that way, other than that being an attempt to make text look fancier.

Reply:
I did not even know that `née` was a word. Thank you @anthony-kolesov for clearing that out.

@VVIsaev Please ignore my original remark.

Reply:
The more I look at this, the more I see that this is an invalid use of that word. The text should be changed, just not to "see". The text under review also changes the meaning of the phrase: it was that "libgloss is very specific to your board, therefore we don't provide real implementations for your board, just a generic thing for nSIM and a null implementation", new meaning is something like "libgloss is very specific to your board, but we provide implementations to cover various cases". However libhl in fact board-agnostic and can work with either an nSIM or xCAM or HW+MDB, so the original reference to "boards" becomes not so relevant. I think the whole paragraph should be something like:

```suggestion
Libgloss implementation is often specific to a particular chip and runtime
configuration and it is not possible to cover them all in the toolchain
distribution. However, Synopsys provides several libgloss implementations
to cover as much usecases as possible.
```

(Does GitHub allow to make change suggestions only to a single line?...)

Reply:
done, thank you!